publications-2015.bib

```diff
@@ -267,7 +267,7 @@ @article{2015_24
 }
 
 @article{2015_25,
-   author  = {T. Carraro and C. Goll and A. Marciniak-Czochra and A. Mikelic´},
```

Comment:
i think this is supposed to be just `\'{c}`, see https://arxiv.org/abs/1401.6278

Reply:
but to be fair, i don' know how `\'{c}'` renders...

Reply:
Oh yeah, thats a mistake... My editor placed an extra character there. Will fix.